ttnn/cpp/ttnn/operations/normalization/rmsnorm_distributed/rmsnorm_distributed_pybind.cpp

```diff
@@ -0,0 +1,58 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "rmsnorm_distributed_pybind.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+#include "rmsnorm_pre_all_gather.hpp"
+#include "rmsnorm_post_all_gather.hpp"
+
+namespace ttnn::operations::normalization::detail {
+
+namespace py = pybind11;
+
+void bind_normalization_rmsnorm_pre_all_gather_operation(py::module& module) {
+
+    ttnn::bind_registered_operation(
+        module,
+        ttnn::rms_norm_pre_all_gather,
+        R"doc(rms_norm_pre_all_gather(input_tensor: ttnn.Tensor, output_dtype: Optional[ttnn.DataType] = None) -> ttnn.Tensor
+            Compute sum(:attr:`input_tensor`ˆ2) and sum(:attr:`input_tensor`) over the last dimension.
+        )doc",
+        ttnn::pybind_arguments_t{
+            py::arg("input_tensor"),
+            py::kw_only(),
+            py::arg("output_dtype") = DataType::BFLOAT16,
```

Comment:
ttnn convention uses 
* `dtype`  (instead of output_dtype)
* `memory_config` (instead of output_mem_config)

Please update. Unfortunately you also have to manually find all usages of output_dtype in py  models/tests and replace them. But I think this op is not tooooo widely used.

Thank you!